sklearn/linear_model/ridge.py

```diff
@@ -1352,10 +1359,10 @@ def _svd_decompose_design_matrix(self, X, y, sqrt_sw):
 
     def _solve_svd_design_matrix(
             self, alpha, y, sqrt_sw, X_mean, v, U, UT_y):
```

Comment:
here and in `_svd_decompose_design_matrix` maybe `v` should also be called `eigvals` ?

Reply:
I called it `eigenvals_sq` just to denote it it `S**2`.

Reply:
they are the eigenvalues of X^TX, ie the squared singular values of X, so I guess it should be `eigenvals` or `singvals_sq`?

Reply:
When performing SVD, we do not explicitly calculate `X^TX`. When one reads the code, it explicit shows `s` is the eigenvalues of X and `eigenvals_sq` is the square of it.

If I use `eigenvals = s**2`, a future reviewer would wonder "why are we calling the eigevals the square of the actual eigenvalues?". (It is unclear from the code/comments/references that we went the eigenvals of X^TX) If we following the references directly, the idea of covariance is not stated anywhere while reading section 5.2. It is briefly mentioned in sectino 5.3.

Reply:
then how about `singvals_sq` or `squared_singvals` or `squared_singular_values`?  because they are the singular values of `X`

Reply:
> It is briefly mentioned in sectino 5.3.

section 5.3 is about the case where we don't want the LOOE and use a separate validation set; although it also computes an eigendecomposition of X^TX  it suggests something quite different and I don't think we should mention it at all

Reply:
I think `singvals_sq` works.